browser/sources.gni

```diff
@@ -253,6 +253,16 @@ if (enable_brave_vpn) {
     "//brave/components/brave_vpn/browser/connection:api",
     "//brave/components/brave_vpn/common",
   ]
+
+  if (is_win) {
```

Comment:
with the suggestions above this could be replaced with `//brave/components/brave_vpn/browser/connection` and no os guards